README.md

```diff
@@ -15,7 +15,7 @@ The rules can be run manually or automatically by a job which is based on OpenMR
   * The following is a sample insert command.
 
 ```sql
-insert into dataintegrity_rule(rule_name,rule_category,handler_config,handler_classname) values('Questionable date of death','patient','java','org.openmrs.module.dataintegrity.InvalidDateOfDeath');
+insert into dataintegrity_rule(rule_name,rule_category,handler_config,handler_classname) values('Questionable date of death','patient','java','org.openmrs.module.basicexample.dataintegrity.InvalidDateOfDeath');
```

Comment:
@ssmusoke , we host these rules inside any module and if we create module using openmrs-sdk then default module name is `basicexample`. It is likely that anyone trying dataintegrity module will keep the default module name, so that's why I put it there. I can add a note regarding this.